roundware/api2/serializers.py

```diff
@@ -342,10 +342,16 @@ def to_representation(self, obj):
 
 
 class SpeakerSerializer(serializers.ModelSerializer):
```

Comment:
worth adding a test of this in the api tests somewhere

Reply:
Do you mean write tests that validate that the serializer is outputting everything in the expected way via the API? I don't think I have an example of this to crib from, but good idea for sure...